### PR TITLE
[ISSUE #292]Translation for "Best Practices-权限控制" Section in the v4.x & 5.0 Document (CN -> EN)

### LINKED ISSUE #292: Translation for "Best Practices-权限控制" Section in the v4.x & 5.0 Document (CN -> EN)
The Chinese version of the "Best Practices-权限控制" section in the v4.x & v5.0 document has now been finished. However, its English version is still incomplete.

We now require someone who knows both Chinese and English and understands the technical terms of Apache RocketMQ to build it. The translation work is not only copy/paste from translation tools but also requires you to have some basic knowledge of Apache RocketMQ and technical writing.

### PR DESCRIPTION
**Pull request by mxsm**
Please do not create a Pull Request without creating an issue first. 

## What is the purpose of the change

close #292 

## Brief changelog

- Translation for "Best Practices-权限控制" Section in the v4.x & 5.0 Document (CN -> EN)

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [ ] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: i18n/en/docusaurus-plugin-content-docs/current/05-bestPractice/18access.md [hunk 1]
```diff
@@ -1,38 +1,48 @@
...
-
-具体可以参考**distribution/conf/plain_acl.yml**配置文件
-
-## 3. 支持权限控制的集群部署
-在**distribution/conf/plain_acl.yml**配置文件中按照上述说明定义好权限属性后，打开**aclEnable**开关变量即可开启RocketMQ集群的ACL特性。这里贴出Broker端开启ACL特性的properties配置文件内容：
+# Permission control
+
+## 1. Introduction to permission control features
+
+Permission control (ACL) mainly provides advanced access control functions at the Topic resource level for RocketMQ. When using RocketMQ permission control, users can inject user name and password parameters into the Client client to achieve signature, and the server can implement permission management and verification of various resources through permission control parameters.
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/05-bestPractice/18access.md]: There are duplicate word usages of "Client" in this sentence.

**Review by tsunghanjacktsai — CHANGES_REQUESTED** [review]
Hey @mxsm ,

The overall looks good to me, but I believe the word "Access Control" fits the "ACL" meaning better than "Permission Control" here. So I'd suggest that replace some of the "permission" with "access" here.

File: i18n/en/docusaurus-plugin-content-docs/current/05-bestPractice/18access.md (hunk 1 above)
Comment (mxsm) [reply i18n/en/docusaurus-plugin-content-docs/current/05-bestPractice/18access.md]: @tsunghanjacktsai  got it, i will fix it later

**Comment by mxsm** [comment]
@tsunghanjacktsai  PTAL~

**Review by tsunghanjacktsai — APPROVED** [review]

**Review by RongtongJin — APPROVED** [review]

**Pull request merged**